identityClient/api.go

```diff
@@ -688,3 +688,22 @@ type ListRealmApplicationMappersRequest struct {
 type ListRealmApplicationMappersResponse struct {
 	ApplicationMappers []ApplicationMapper `json:"application_mappers"`
 }
+
+// SearchRealmIdentitiesResponse wraps matching identities in the given realm for the search criteria (if any)
+type SearchRealmIdentitiesResponse struct {
+	Identities map[uuid.UUID]string `json:"identities"`
+	NextToken  int64                `json:"next_token"`
+}
```

Comment:
```go
// SearchRealmIdentitiesResponse wraps matching identities in the given realm for the search criteria (if any)
type SearchRealmIdentitiesResponse struct {
	Identities                          []Identity                `json:"identities"`
	IdentitiesEmailToToznyClientAliases EmailToToznyClientAliases `identity_email_to_client_aliases`
	NextToken                           int64                     `json:"next_token"`
}

type EmailToToznyClientAliases map[string]uuid.UUID
```

Reply:
(we'll want the map key to be an email because that is what the user will input into the system and what the frontend will want to look up off of